base.orogen

```diff
@@ -30,16 +29,19 @@ import_types_from "base/JointLimits.hpp"
 import_types_from "base/JointTransform.hpp"
 import_types_from "base/JointsTrajectory.hpp"
 
-if has_library?('base-lib')
+sisl_available = has_library?('base-lib-sisl')
+
+if sisl_available
```

Comment:
This is inconsistent with the next line. It's either both `base-lib` or both `base-lib-sisl`.

Reply:
ack

Reply:
changed it to base-types-sisl